CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ set(PATCH_VERSION 0)
 set(SYSSTAT_VERSION "${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}")
 
 set(LXQTBT_MINIMUM_VERSION "0.4.0")
-set(QT_MINIMUM_VERSION "5.4.2")
+set(QT_MINIMUM_VERSION "5.7.1")
```

Comment:
The minimum qt version is enforced by the `lxqt-build-tools`... so we should drop it here and require the 0.4.1 of build tools (if we agree, that the `lxqt-build-tools` patch version should be increased with the Qt v5.7.1)...